api/v1/helpers/nouns/users.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
     POST: `Create a new ${m}`,
     PUT: `Overwrite all attributes of this ${m}`,
   },
+  scopeMap: {
+    withoutSensitiveInfo: 'withoutSensitiveInfo',
```

Comment:
I think we should leave it for now and make sure the other story gets prioritized.